backend/models/orderModel.js

```diff
@@ -34,6 +34,7 @@ const orderSchema = new mongoose.Schema(
     taxPrice: { type: Number, required: true },
     totalPrice: { type: Number, required: true },
     user: { type: mongoose.Schema.Types.ObjectId, ref: 'User', required: true },
+    seller: { type: mongoose.Schema.Types.ObjectID, ref: 'User' },
```

Comment:
ObjectID should be ObjectId?, can this cause the error with {product.seller.seller.name} call in Products window?

Reply:
in productModel.js have the same ObjectID, is necessary remplace to ObjectId. but in data.js verify if you have this:
users: [    {     
name: 'Basir',      
email: 'admin@example.com',      
password: bcrypt.hashSync('1234', 8),      
isAdmin: true,      
isSeller: true,      
seller: {        
name: 'Puma',        
logo: '/images/logo1.png',        
description: 'best seller',        
rating: 4.5,        
numReviews: 120,      
},   
 },

Reply:
fher help me!! ... why now, a normal user only can buy admin's products ??
it is a issue that I having and I dont now why ... 
The consoles says me:
POST http://localhost:3000/api/orders 500 (Internal Server Error)
I thought it would be an error of OrderRouter but I have the same code as Basir ....